src/app/App.test.tsx

```diff
@@ -2,8 +2,25 @@ import React from "react";
 import { render } from "@testing-library/react";
 import App from "./App";
 
+type MockEnvironment = {
+  [keyname: string]: string;
+};
+
+afterEach(() => {
+  delete (process.env as MockEnvironment).PUBLIC_URL;
+});
+
 test("Renders WebCore Capability Run Books section", () => {
   const { getByText } = render(<App />);
   const headerElement = getByText(/WebCore Capability Run Books/i);
   expect(headerElement).toBeInTheDocument();
 });
+
+test("Renders a link to performance report using the public URL", () => {
+  (process.env as MockEnvironment).PUBLIC_URL = "https://www.google.com/";
+  const { container } = render(<App />);
+  const reportLink = container.querySelector(
+    "a[href='https://www.google.com/webcore-performance-report-august-v2.pdf']"
```

Comment:
Does querySelector support the contains `*=` operator, we could drop the google.com stuff that way. 

[Answered my own question, yes it does. ](https://drafts.csswg.org/selectors/#attribute-substrings)

```suggestion
    "a[href*='/webcore-performance-report-august-v2.pdf']"
```

Also, might be useful to just put the relative path to the doc so not including the `PUBLIC_URL` bit.

Reply:
If we did that we'd no longer be asserting on the thing we are actually testing, James - which is that the URL is constructed from the value that we assigned to `process.env.PUBLIC_URL`

Reply:
> Also, might be useful to just put the relative path to the doc so not including the PUBLIC_URL bit.

This is the approach recommended by the CRA documentation